main.py

```diff
@@ -72,19 +78,19 @@ def create_climb():
             
         def save_climb():
             self.climb_save.handle_save_climb()
+            print(self.climb.route)
```

Comment:
climb is a string during runtime somehow

Reply:
In the main branch, it does actually print the route as a list after you create an instance of the climb with the create climb button. But when I was trying to fix collect_route and also add functionality to the save climb button something got messed up and I'm not sure where!